locales/templates/app.pot

```diff
@@ -1446,6 +1446,47 @@ msgctxt "logo_text"
 msgid "Service provided by"
 msgstr ""
 
+#. Recommended news onboarding title and content. The %1$s here is a
+#. placeholder for "Learn more" which directs users to SUMO page with detail
+#. information about the feature.
+msgctxt "recommended_news_title"
+msgid "Recommended News For You"
```

Comment:
I don't see the placeholder in the string anymore, am I missing something?

Reply:
I guess they messed up with previous one. Will check with UX